graphql-tool-idl-plugin/src/test/java/io/ballerina/graphql/idl/client/IDLClientGenPluginTests.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.graphql.idl.client;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static io.ballerina.graphql.idl.client.TestUtils.getMatchingFiles;
+
+/**
+ * Client IDL import integration tests.
+ */
+public class IDLClientGenPluginTests extends GraphqlIDLTest {
+
+    @Test(description = "Single client declaration in module level")
+    public void testSingleClientDeclaration() throws IOException, InterruptedException {
+        File[] matchingFiles = getMatchingFiles("project_01");
+        Assert.assertNotNull(matchingFiles);
+        Assert.assertEquals(matchingFiles.length, 1);
+    }
```

Comment:
Let's use the java temp directory to copy test resources and execute the test cases, instead of using the original test resources directory